src/gluonts/model/deepar/_network.py

```diff
@@ -39,6 +40,44 @@ def prod(xs):
     return p
 
 
+def mx_switch(F, *args, **kwargs):
+    """
+    A switch statement for mxnet
+
```

Comment:
Shouldn't we move mx_switch to mx/utils.py? It will be clean in a case other models use it.

Reply:
Done.